<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Borrow</class>
 <widget class="QWidget" name="Borrow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>221</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDockWidget" name="dockWidget">
      <widget class="QWidget" name="dockWidgetContents">
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>221</width>
          <height>441</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Novel</string>
         </property>
         <item>
          <property name="text">
           <string>Science fiction</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Romantic fiction</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>History fiction</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Speculative Fiction</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Learning Materials</string>
         </property>
         <item>
          <property name="text">
           <string>Chinese</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Math</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Physics</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Periodical</string>
         </property>
         <item>
          <property name="text">
           <string>Computer Science</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Psychology</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chemistry</string>
          </property>
         </item>
        </item>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>511</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="led_search">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Search">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Name :</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>59</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/qss/psblack/branch_open.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>230</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>The Cover</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Author Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>400</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_book">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>265</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_author">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_ISBN">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>390</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Borrow">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
